.gitlab-ci.yml

```diff
@@ -0,0 +1,47 @@
+include:
+  local: templates/v6.yml
+
+.install:
+  script: |
+    apt-get -qq update
+    DEBIAN_FRONTEND=noninteractive apt-get -qqy -o=Dpkg::Use-Pty=0 install wget
+    eval "$(bash install_julia.sh $JULIA)"
+    hash julia 2> /dev/null
+
+Install Julia 1.0:
```

Comment:
why does this use a manual target that invokes the script? shouldn't this just `extends: .julia: 1.0` as users will?

Reply:
You're right, it should just extend the existing target and then test that the installation worked in `script`.